.github/workflows/build.yml

```diff
@@ -154,61 +128,45 @@ jobs:
         run: echo "::set-env name=BRANCH::${GIT_BRANCH}"
 
       - name: Security audit Node modules
-        run: yarn run security-audit
+        run: ./scripts/security-audit.sh
```

Comment:
why create scripts for one liners?

Reply:
Longer discussion, let's have it "live" ;)